WD-Firmware/examples/build/SConscript_bsp_hifive1

```diff
@@ -37,13 +37,16 @@ c_flags = [] + Env['C_FLAGS']
 a_flags = [] + Env['A_FLAGS']
 
 # defines

```

Comment:
write : "compilation defines" (D_)
move all defines, linker, compiler, etc.... to the beginning of the file